src/masquerade/providers/web_api.py

```diff
@@ -52,10 +52,10 @@ def etl_candidate(agrc_candidate):
     return {
         'address': address,
         'attributes': {
-            'score': agrc_candidate['score'],
+            'Score': agrc_candidate['score'],
             'locator': agrc_candidate['locator'],
             'addressGrid': agrc_candidate['addressGrid']
```

Comment:
**question**: I wonder if `locator` and `addressGrid` should be title-cased as well just for consistency? Same question for `address`, `attributes`, & `location` in the parent object. What does the Esri locator service return object look like?

Reply:
I'll try to take a look later this week to see what I can figure out.